app/assets/javascripts/notes.js

```diff
@@ -0,0 +1,11 @@
+$(() => {
```

Comment:
It is better to use classes rather than methods. But you can leave it as it is if you want.